marklogic-data-hub/src/main/java/com/marklogic/hub/HubConfig.java

```diff
@@ -26,10 +26,11 @@
 import javax.net.ssl.SSLContext;
 import javax.net.ssl.X509TrustManager;
 import java.nio.file.Path;
-import java.util.Properties;
 
 /**
- * An interface to set, manage and recall the Data Hub's Configuration
+ * An interface to set, manage and recall the Data Hub's Configuration.
+ * HubConfig has a singleton scope so set everything you want at the start of the application and
+ * and then call {@link #refreshProject()} to wire up all clients.
```

Comment:
should "then call {@link #refreshProject()} to wire up all clients." be "then call {@link #refreshProject()} to load the properties from gradle.properties (optionally overridden with gradle-{env}.properties) and wire up all clients ?

Reply:
Updated here https://github.com/marklogic/marklogic-data-hub/pull/1683/commits/cff6c869b2e42f0179807f07809c7402517c0c35